src/game_renderer/GameRenderer.cpp

```diff
@@ -69,10 +69,14 @@ void GameRenderer::Init(const std::string& resource_dir,
   glfwWindowHint(GLFW_OPENGL_PROFILE, GLFW_OPENGL_CORE_PROFILE);
   glfwWindowHint(GLFW_CONTEXT_VERSION_MAJOR, 3);
   glfwWindowHint(GLFW_CONTEXT_VERSION_MINOR, 2);
-  window = glfwCreateWindow(WINDOW_WIDTH, WINDOW_HEIGHT, TITLE, NULL, NULL);
+
+  GLFWmonitor* monitor = glfwGetPrimaryMonitor();
+  const GLFWvidmode* mode = glfwGetVideoMode(monitor);
+  window = glfwCreateWindow(mode->width, mode->height, TITLE, monitor, NULL);
```

Comment:
Haven't messed with that yet. If we want it to be "fair" in that people with wider monitors don't get to see further ahead, I'll make a fix for that later on